eng/new-issue.ps1

```diff
@@ -0,0 +1,76 @@
+### Creates an issue, assigns it to the PCS and UB projects, sets the area/size fields and opens the issue in the browser.
```

Comment:
should we add a switch for the UB project? not all issues will be there? But maybe we just create it manually in that case, like we would for FR

Reply:
Yeah, why not